apps/desktop-wallet/src/features/thumbnails/NFTThumbnail.tsx

```diff
@@ -17,18 +17,18 @@ along with the library. If not, see <http://www.gnu.org/licenses/>.
 */
 
 import { NFT } from '@alephium/shared'
-import { X } from 'lucide-react'
-import { memo } from 'react'
+import { CameraOff } from 'lucide-react'
+import { memo, useState } from 'react'
 import { useTranslation } from 'react-i18next'
-import styled, { useTheme } from 'styled-components'
+import styled from 'styled-components'
 
 import useFetchNft from '@/api/apiDataHooks/token/useFetchNft'
-import Spinner from '@/components/Spinner'
+import SkeletonLoader from '@/components/SkeletonLoader'
 import VideoThumbnail from '@/features/thumbnails/VideoThumbnail'
 
 interface NFTThumbnailProps {
   nftId: NFT['id']
-  size?: number | string
```

Comment:
We never use `number` (and it also causes type issues with the Skeleton loader). We can change it in the future if we need number.